public/app/plugins/datasource/loki/components/LokiContextUi.tsx

```diff
@@ -136,108 +147,129 @@ export function LokiContextUi(props: LokiContextUiProps) {
   const parsedLabels = contextFilters.filter(({ fromParser }) => fromParser);
   const parsedLabelsEnabled = parsedLabels.filter(({ enabled }) => enabled);
 
+  const contextFilterToSelectFilter = useCallback((contextFilter: ContextFilter): SelectableValue<string> => {
+    return {
+      label: `${contextFilter.label}="${contextFilter.value}"`,
```

Comment:
I think we need to escape the value here using `escapeLabelValueInSelector`:
<img width="169" alt="image" src="https://user-images.githubusercontent.com/8092184/232737231-aa54d078-a8ef-481a-851e-3acf46a92c91.png">

Reply:
Good catch! Fixed in https://github.com/grafana/grafana/pull/66730/commits/5af191b96aeec35e28b6aad44059a8e36791bc63